### ISSUE TITLE
Wrong results with `no precompilation;`

### ISSUE THREAD

**Issue by lukasvalle** [vrurg/perl6-IP-Addr#4]
The module does seems only to work correct, if the code was precompiled. 
Normal behavior as expected:
```
raku -v
This is Rakudo version 2019.11 built on MoarVM version 2019.11
implementing Perl 6.d.

RAKUDOLIB=./lib/ raku -MIP::Addr -e 'IP::Addr.new("1.1.1.1").say'
#1.1.1.1
```

To disable precompilation I've added  `no precompilation;` in the  [Handler.pm6](https://github.com/vrurg/perl6-IP-Addr/blob/master/lib/IP/Addr/Handler.pm6):
```
#! /usr/bin/env false

use v6.c;
no precompilation;
```
Then the module does return `0.0.0.0` instead of `1.1.1.1` and multiple warnings are shown on the console:
```
RAKUDOLIB=./lib/ raku -MIP::Addr -e 'IP::Addr.new("1.1.1.1").say'
Use of uninitialized value of type Any in numeric context
  in method bitcap at -e line 106
Use of uninitialized value of type Any in numeric context
  in method bitcap at -e line 106
Use of uninitialized value of type Any in numeric context
  in method bits2mask at -e line 322
...
Use of uninitialized value of type Any in numeric context
  in method bitcap at -e line 106
0.0.0.0
```

It does look like, that the [%bitcap-mask Hash](https://github.com/vrurg/perl6-IP-Addr/blob/master/lib/IP/Addr/Handler.pm6#L36) only can be read correctly when precompiled before. If I change the `%bitcap-mask` to an our variable, the code does work also without precompilation:
```
our %bitcap-mask = (0..128).map: { $_ => 2**$_ - 1 }; # Mask for each bit capacity
```
It does also work, if I place the `%bitcap-mask`  after the [unit role initialisation](https://github.com/vrurg/perl6-IP-Addr/blob/master/lib/IP/Addr/Handler.pm6#L38)
```
#my %bitcap-mask = (0..128).map: { $_ => 2**$_ - 1 }; # Mask for each bit capacity

unit role IP::Addr::Handler;
my %bitcap-mask = (0..128).map: { $_ => 2**$_ - 1 }; # Mask for each bit capacity
```
I would have expected, that the precompilation does only affect startup speed, but not the behavior of the code.

**Comment by vrurg** [comment]
@lukasvalle thanks for spotting this! 

> I would have expected, that the precompilation does only affect startup speed, but not the behavior of the code.

Perhaps, not in this case. I presume that in this case what should've happened is compiler reporting an error because the only statements allowed before `unit` must be `use` and pragmas.

I will move the declaration and re-publish the module soon.

**Comment by vrurg** [comment]
Fixed in v0.0.3

**Comment by vrurg** [comment]
@lukasvalle Thank you!
